addons/calendar/static/src/activity/activity_patch.xml

```diff
@@ -10,6 +10,30 @@
                 <i class="fa fa-calendar"/> Reschedule
             </button>
         </xpath>
+        <xpath expr="//div[hasclass('o-mail-Activity-iconContainer')]" position="attributes">
+            <attribute name="t-att-class">
```

Comment:
You should add an empty `t-att-class` on the parent element to be overridden here. To avoid any future breaks.

Reply:
I think `t-att-class` already present on the parent element.

Reply:
Then you should not overwrite the attribute this way. As it makes it tough to maintain.